uint8 prn

uint8 trackedL1
uint8 trackedL2
uint8 trackedAtBaseL1
uint8 trackedAtBaseL2
uint8 usedInSln
uint8 usedInRTK
uint8 trackedPcodeL1
uint8 trackedPcodeL2

float32 elevation
float32 azimuth
float32 snrL1
float32 snrL2
